docs/index.md

```diff
@@ -26,6 +26,11 @@ managing them. Using these commands you can
  - upgrade the Docker client and daemon
  - configure a Docker client to talk to your host
 
+## Install
```

Comment:
I would tend to leave the heading off all together as the sentence would come at the end of the bullet list in the intro section --- it gives enough emphasis.  If you don't like that, I"d tweak the heading level and the head itself so as not to imply these are instructions which the command 

`#### Need the installation instructions?`